setup.cfg

```diff
@@ -57,6 +57,8 @@ scripts =
 [options.extras_require]
 scikit-image =
     scikit-image
+specutils =
+    specutils
```

Comment:
This installs `specutils` when PypeIt is installed via `pip`... is there a way to install optional packages when installing via `conda`?

The [current documentation](https://pypeit.readthedocs.io/en/develop/installing.html) states that installing via `conda` is the preferred method, and the only way for Apple Silicon.

If there is not a way to optionally install packages, then the `environment.yaml` file needs to be amended to add `specutils`.

Reply:
Thanks to both of you!  I added a the `specutils` dependency in the `environment.yml` file, in the `pip` section.  Does it look right?

Reply:
PS - I'm also happy to squeeze in some alterations to our installing doc, based on your recommendations, @tepickering .